contract/src/proxy.rs

```diff
@@ -3,30 +3,34 @@
 
 extern crate alloc;
 
-use alloc::boxed::Box;
-use alloc::string::String;
-use alloc::{format, vec};
-use alloc::{string::ToString, vec::Vec};
-use casper_contract::contract_api::runtime::call_versioned_contract;
-use casper_contract::contract_api::storage::{dictionary_get, dictionary_put};
+use alloc::{
+    boxed::Box,
+    string::{String, ToString},
+    vec::Vec,
+    {format, vec},
+};
 use casper_contract::{
     contract_api::{
-        runtime::{self, revert},
-        storage::{self, new_dictionary},
+        runtime::{self, call_versioned_contract, revert},
+        storage::{self, dictionary_get, dictionary_put, new_dictionary},
     },
     unwrap_or_revert::UnwrapOrRevert,
 };
 use casper_types::{
-    contracts::NamedKeys, ApiError, CLType, CLTyped, ContractPackageHash, EntryPoint,
-    EntryPointAccess, EntryPointType, EntryPoints, Key, Parameter, URef,
+    contracts::NamedKeys, runtime_args, ApiError, CLType, CLTyped, CLValue, ContractPackageHash,
+    EntryPoint, EntryPointAccess, EntryPointType, EntryPoints, Key, Parameter, RuntimeArgs, URef,
+    U256,
 };
-use casper_types::{runtime_args, CLValue, RuntimeArgs, U256};
 
+/// EntryPoint that call the initializes the contracts storage. This entry_point can only be called once,
```

Comment:
```suggestion
/// EntryPoint that calls the initialized contract storage. This entry_point can only be called once,
```

Reply:
The contract is responsible for initialising the contracts storage. Although the sentence does look bad and I can understand the confusion.